http.go

```diff
@@ -270,6 +272,22 @@ func (hc *HttpCall) ResponseBodyContains(value string) Step {
 	})
 }
 
+// ResponseBodyMatches is a Step that when executed ensures there is
+// a non-nil hc.ResponseBody and errors unless the hc.ResponseBody
+// matches the regular expression parameter.
+func (hc *HttpCall) ResponseBodyMatches(pattern string) Step {
+	return NewNamedStep("ResponseBodyMatches", func() error {
+		if err := hc.ReceiveBody(); err != nil {
+			return err
+		}
+		var regex = regexp.MustCompile(pattern)
+		if !regex.MatchString(string(hc.ResponseBody)) {
+			return fmt.Errorf("Body: Expected to match the pattern '%s'; found '%s'.", pattern, string(hc.ResponseBody))
```

Comment:
*Regexp does indeed have a sensible String() method, so this error makes a lot of sense. But that also means that in the first arg to NewNamedStep you can use fmt.Sprintf and have a more useful name - again in keeping with all the other steps.